integration/ct_hammer/main.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/google/certificate-transparency/go/x509"
+	ctfe "github.com/google/trillian/examples/ct"
+	"github.com/google/trillian/integration"
+)
+
+var (
+	httpServersFlag = flag.String("ct_http_servers", "localhost:8092", "Comma-separated list of (assumed interchangeable) servers, each as address:port")
+	testDir         = flag.String("testdata", "testdata", "Name of directory with test data")
+	seed            = flag.Int64("seed", -1, "Seed for random number generation")
+	logConfigFlag   = flag.String("log_config", "", "File holding log config in JSON")
+	mmdFlag         = flag.Duration("mmd", 2*time.Minute, "MMD for logs")
+	operationsFlag  = flag.Uint64("operations", ^uint64(0), "Number of operations to perform")
+)
+var (
+	addChainBias          = flag.Int("add_chain", 20, "Bias for add-chain operations")
+	addPreChainBias       = flag.Int("add_pre_chain", 20, "Bias for add-pre-chain operations")
+	getSTHBias            = flag.Int("get_sth", 2, "Bias for get-sth operations")
+	getSTHConsistencyBias = flag.Int("get_sth_consistency", 2, "Bias for get-sth-consistency operations")
+	getProofByHashBias    = flag.Int("get_proof_by_hash", 2, "Bias for get-proof-by-hash operations")
+	getEntriesBias        = flag.Int("get_entries", 2, "Bias for get-entries operations")
+	getRootsBias          = flag.Int("get_roots", 1, "Bias for get-roots operations")
+	getEntryAndProofBias  = flag.Int("get_entry_and_proof", 0, "Bias for get-entry-and-proof operations")
+)
+
+func main() {
+	flag.Parse()
+	if *logConfigFlag == "" {
+		glog.Fatal("Test aborted as no log config provided")
```

Comment:
That doesn't seem to work - is there some magic way of getting the flag name back?  (`logConfigFlag` appears to be a `*string`).

(Added explicit text instead).